samples/subsys/lorawan/class_a/src/main.c

```diff
@@ -64,9 +63,9 @@ void main(void)
 		.cb = dl_callback
 	};
 
-	lora_dev = device_get_binding(DEFAULT_RADIO);
-	if (!lora_dev) {
-		LOG_ERR("%s Device not found", DEFAULT_RADIO);
+	lora_dev = DEVICE_DT_GET(DEFAULT_RADIO_NODE);
```

Comment:
we could probably just do:

```suggestion
	lora_dev = DEVICE_DT_GET(DT_ALIAS(lora0));
```

and remove the rest.